kas/base.yml

```diff
@@ -44,6 +44,10 @@ repos:
         path: "layers/openembedded-core"
         layers:
             meta:
+        patches:
+            patch0:
+                repo: meta-nile
+                path: ../kas/patches/oe-core/0001-Revert-lib-oe-go-document-map_arch-and-raise-an-erro.patch
```

Comment:
*[Just a comment. Not blocking this PR]*

It's nice that we don't have to create a fork of the repo to revert the commit.
But I suppose when upstream fixes it, this patch will cause build failures right?

In past we disabled nile builds in ni-central during release time so release doesn't get affected by nile build failures but we need a longer term solution.

@bstreiff @erickshepherdNI @gratian